src/Exception/InvalidConfigException.php

```diff
@@ -5,11 +5,16 @@
 use LogicException;
 use TomPHP\ExceptionConstructorTools;
 
+/**
+ * @api
```

Comment:
@localheinz I've removed the `@api` tags but left the `@internal` tags on the exception constructors - I just don't want to be BC locked against them because no one should be called them externally.

Once this is done it's `1.0.0` time :-D